fineract-provider/src/main/java/org/apache/fineract/portfolio/self/pockets/service/AccountEntityServiceForShareAccountsImpl.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.fineract.portfolio.self.pockets.service;
+
+import org.apache.fineract.infrastructure.accountnumberformat.domain.EntityAccountType;
+import org.apache.fineract.infrastructure.security.service.PlatformSecurityContext;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AccountEntityServiceForShareAccountsImpl implements AccountEntityService {
+
+	private final String KEY = EntityAccountType.SHARES.name();
+
+	private final PlatformSecurityContext context;
+
+	@Autowired
+	public AccountEntityServiceForShareAccountsImpl(PlatformSecurityContext context) {
+		this.context = context;
+	}
+
+	@Override
+	public String getKey() {
+		// TODO Auto-generated method stub
```

Comment:
This class looks un-finished... not sure it's a good idea to merge this with empty TODO method bodies? If perhaps you would consider removing this, then I think the rest could probably be merged (I would not mind to merge it).

Reply:
@vorburger  , will update it soon Thanks for reviewing